libs/components/src/lib/data-grid/data-grid-cell.ts

```diff
@@ -327,28 +333,94 @@ export class DataGridCell extends Localized(VividElement) {
 		}
 	};
 
+	/**
+	 *
+	 * @private
+	 */
+	private calculateAriaSelectedValue() {
+		if (this.selectable && this.selected) return 'true';
+
+		if (this.selectable && !this.selected) return 'false';
+
+		return null;
+	}
+
+	/**
+	 * @internal
+	 */
+	@observable
+	selectable = false;
```

Comment:
Should have an underscore as well:
```suggestion
	_selectable = false;
```